documentation/airport-file-standards.md

```diff
@@ -0,0 +1,89 @@
+"LACEE": ["N47d02.82m0", "W122d48.35m0"],
+"LIINE": ["N46d50.65m0", "W122d40.11m0"],
+"LKV"  : ["N42d29.57m0", "W120d30.42m0"],
+"MARNR": ["N48d02.33m0", "W122d50.99m0"]
+```
+
+For custom RNAV fixes, such as `_RWY22L05DME`, these shall use single spaces.
+
+### Bracket Newlines
+To maintain uniformity in all airport files, please follow the same newline conventions as the example file shown in the [airport format](./airport-format.html) document.
+
+### Key order
+To maintain uniformity in all airport files, all keys used in the file must be included in the same order as they appear in the [airport format](./airport-format.html) document.
+- All fixes must be sorted alphabetically.
+- The first object within the `airspace` array must represent the airspace's _outer perimeter_, which encompasses all other airspace sections.
+- All runways should be sorted numerically (then L-C-R) by the primary end
+- All SIDs/STARs must be sorted alphabetically by ID
+- Spawn patterns should have _departure_ patterns, then _arrival_ patterns, then any _overflight_ patterns.
+
+Please note that spawnPatterns must include all keys, even if they are not used.
+
+---
+
+## Requirements for "Premium Airport" Designation
+### Airways
+Relevant airways, whether they are used by procedures or spawn patterns or not, must be included as a part of the airport submission. This includes:
+- All airways within the lateral airspace boundary
+    - Must extend a minimum of 50 miles from the airspace boundary
+- All airways _connected to_ any airway that transits the airspace
+    - Must extend a minimum of 50 miles from the airspace boundary
+
+### Full documentation on facility's airspace
+Depending on the country of origin, this can be difficult to obtain or generate. But to the extent possible, documentation must be generated about the general flows of traffic, relevant local procedures and conventions, standard/acceptable/unacceptable routings, agreements with neighboring sectors or facilities, and any other unusual information specific to this facility that a user would need to know in order to run the airspace realistically.
+
+### Airspace and traffic flow diagrams
+To go with the written documentation, diagrams should be provided as a visual aid to show the airspace structure, alignment of SID/STARs, arrival and departure corridors (if applicable), airspace sectorization (including what "sector(s)" _we_ are simulating).
+
+### Accurate Airspace Stratification
+The airspace in the submitted airport must have the same stratification (aka 'shelving') as the real-world facility. This means that if one section is owned from the SFC-8000, and another section from 4000-8000, they must have separate entries within the `airspace` tag of the airport file. Users will be able to examine the airspace stratification _in-game_ with a button that will toggle on and off the airspace section outlines with altitude ranges indicated within them in text.
```

Comment:
Honestly, I'm not completely sure. I think not? I remember that the _first_ entry within `airspace` is assumed to be the "perimeter", and I think it will actually parse all other objects within `airspace`, though they are not used for anything at the moment.